playwright-e2e/dss/component/checkbox.ts

```diff
@@ -30,6 +30,11 @@ export default class Checkbox extends WidgetBase {
   }
 
   async isChecked(): Promise<boolean> {
+    try {
+      await this.toLocator().scrollIntoViewIfNeeded();
+    } catch (error) {
+      // ignored
```

Comment:
since the caught error is ignored, would it be better to use an expect.toBeVisible() here?

Reply:
I don't want test to be interrupted or terminated if scrollIntoView has failed. This Playwright API is known to be flaky.